pkg/logql/downstream_test.go

```diff
@@ -508,18 +510,13 @@ func TestSketchEquivalence(t *testing.T) {
 					}
 				}
 
-				// We allow max 2 misses per vector.
-				// TODO: maybe use percentage. Could use InDelta or InEpsilon instead
-				require.LessOrEqual(t, misses, 2)
-
-				totalMisses += misses
+				f.Update(float64(misses))
 
 				ok, ts, expectedVec = expected.Next()
 			}
 
-			// Overall we want to capture 98% of topk.
-			missedPercentage := float64(totalMisses) / float64(totalLabels)
-			require.LessOrEqual(t, missedPercentage, 2.0)
+			require.LessOrEqual(t, f.Quantile(0.99), 2.0)
+			require.LessOrEqual(t, f.Quantile(0.50), 1.0)
```

Comment:
I like this way of checking the accuracy :+1:

Reply:
Thanks. I hope that we have a more refined way tomorrow.